api/Document.json

```diff
@@ -11536,14 +11534,24 @@
               "version_added": "10",
               "notes": "The <code>onvisibilitychange</code> attribute is not supported in IE. To listen to this event, use <code>document.addEventListener('visibilitychange', function() {});</code>."
             },
-            "opera": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
-            },
-            "opera_android": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
-            },
+            "opera": [
+              {
+                "version_added": "12.1"
```

Comment:
If the Chrome data is correct, this should be removed in 15 and added back in 20.